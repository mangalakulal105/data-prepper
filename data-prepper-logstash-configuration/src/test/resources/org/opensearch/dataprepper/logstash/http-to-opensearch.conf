input {
    http {
    }
}
output {
    elasticsearch {
        hosts => ["https://localhost:19000"]
        user => myuser
        password => mypassword
        index => "simple-pipeline"
    }
}