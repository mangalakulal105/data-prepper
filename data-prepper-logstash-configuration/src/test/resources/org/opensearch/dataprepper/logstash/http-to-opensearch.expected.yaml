logstash-converted-pipeline:
  source:
    http:
      max_connection_count: 500
      request_timeout: 10000
  processor: []
  sink:
    - opensearch:
        hosts:
          - "https://localhost:19000"
        username: "myuser"
        password: "mypassword"
        index: "simple-pipeline"
